src/containers/StructurePage/resourceComponents/AddResourceModal.tsx

```diff
@@ -227,7 +227,7 @@ const AddResourceModal = ({
             ? await findResourceIdLearningPath(Number(selected.id))
             : getResourceIdFromPath(selected?.paths?.[0]);
 
-        if (!resourceId) {
+        if (!resourceId || !topicId) {
```

Comment:
Gir dette meining? Det er vel kun ressurser som skal kunne legges til med denne komponenten.

Reply:
`createTopicResource` er avhengig av en topicId som tidligere har vært satt til å være `string | undefined`, men ser nå at den alltid vil være satt. Har fjernet sjekken, men den er for så vidt påkrevd.